docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
++
+[source,shell]
+----
+$ ansible-doc rhoas.rhoas.create_kafka
+----
+. (Optional) You can run a module in the terminal to perform a single task. For example, the following command shows how to run a module that creates a Kafka instance.
++
+.Example create_kafka module
+[source,shell]
+----
+$ ansible localhost -m rhoas.rhoas.create_kafka -a 'name=unique-kafka-name billing_model=standard cloud_provider=aws plan="developer.x1" region="us-east-1" openshift_offline_token=<OFFLINE_TOKEN>'
+----
+
++
+. Open a terminal window.
+. Create a new playbook file and save it as `rhoas_test.yml` in any directory.
+. In your web browser, navigate to https://github.com/redhat-developer/app-services-ansible/blob/main/rhoas_test.yml to open the {product-long-rhoas} example playbook.
+
+. Inspect the contents of the example playbook. In particular, observe that the playbook has modules for the following tasks:
++
+* Creating and deleting a Kafka instance
+* Creating and deleting a service account
+* Creating Access Control List (ACL) permission bindings
+* Creating, updating, and deleting a topic
++
+[NOTE]
+The playbook uses your offline token to authenticate with the Kafka Management API. If the token is not passed in as an argument per task, the module attempts to read it from the `_OFFLINE_TOKEN_` environment variable.
+
++
+[NOTE]
+====
+The example playbook used in this section includes comments that indicate how to directly specify values rather than fetching them dynamically. For example, to specify a Kafka instance ID, a comment in the playbook states that you can include the following line:
+
+[source, subs="+quotes"]
+----
+kafka_id: __<kafka_id>__
+----
+====
++
+. Click on the copy symbol to copy the contents of the example playbook.
+. Open the `rhoas_test.yml` file in your IDE and paste the example into it.
+. Configure the `create_kafka_` module to provision a new Kafka instance. The only required fields for this module are the `name`, `billing_model` and `cloud provider` fields. You can specify configuration options such as `billing_cloud_account_id` and the `openshift_offline_token`. All other information for the instance is provided by the Kafka APIs. The default contents of this module in the OpenShift Application Services example playbook are shown below:
++
+.Example `create_kafka` module
+[source,yaml]
+----
+- name: Create kafka
+    rhoas.rhoas.create_kafka:
+      name: "kafka-name"
+      instance_type: "x1"
+      billing_model: "standard"
+      cloud_provider: "aws"
+      region: "us-east-1"
+      plan: "developer.x1"
+      billing_cloud_account_id: "123456789"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      kafka_req_resp
+----
++
+When you run the `create_kafka_` module, Ansible saves the output of that command in a variable in the `register` field. In the preceding example, Ansible saves the created Kafka instance as `_kafka_req_resp_`.
```

Comment:
I think I actually included this in my previous comment (sorry!), but I'd remove the underscores around `_kafka_req_resp_`. There's no reason for that to be italicized.